_plugins/idp_translations.rb

```diff
@@ -0,0 +1,34 @@
+## This plugin copies the fonts and the images from the identity style guide
+## to the appropriate path on the built site
+##
+## The CSS and JS is already handled by Jekyll and the build:js npm script
+
+require 'fileutils'
+require 'uri'
+require 'net/http'
+require 'json'
+require 'open-uri'
+
+DOMAIN = 'https://secure.login.gov'
+
+def copy_translations(origin_uris, destination)
+  origin_uris.each do |uri|
```

Comment:
We have a small number of assets at the moment, but if the amount grows then I think it may make sense to eventually parallelize the downloads.